crates/forge/tests/e2e/coverage.rs

```diff
@@ -13,3 +18,49 @@ fn test_coverage_project() {
     // Check if it doesn't crash in case some data already exists
     test_runner(&temp).arg("--coverage").assert().success();
 }
+
+#[test]
+fn test_fail_on_scarb_version_lt_2_8_0() {
+    let temp = setup_package("coverage_project");
+
+    let output = test_runner(&temp).arg("--coverage").assert().failure();
+
+    assert_stdout_contains(
+        output,
+        "[ERROR] Coverage generation requires scarb version >= 2.8.0\n",
+    );
+}
```

Comment:
This test will have a different behavior depending on a local scarb setup

Reply:
We have 2.7.0 in tool versions so this should always pass